### ISSUE TITLE
Error: "No such file or directory"

### ISSUE THREAD

**Issue by jmatraszek** [pwoolcoc/cargo-shell#2]
Hi, followed the instructions, but it seems to not work for me:
```
$ cargo build
    Finished debug [unoptimized + debuginfo] target(s) in 0.0 secs
$ cargo shell
Welcome to cargo-shell v0.1.0
>> build
Error: Error { kind: IoError(Error { repr: Os { code: 2, message: "No such file or directory" } }), state: State { next_error: None, backtrace: None } }
```
```
$ rustup --version
rustup 0.6.3
$ rustc --version
rustc 1.13.0 (2c6933acc 2016-11-07)
$ cargo --version
cargo 0.13.0-nightly (eca9e15 2016-11-01)
```

**Comment by pwoolcoc** [comment]
Thanks for opening this. I am busy today because of thanksgiving in the US but I will take a look at this as soon as I can.

**Comment by jmatraszek** [comment]
Hi, I've investigated this a little and found out that this causes problem: `let rustup = Path::new(&env::var("CARGO_HOME")?).join("bin").join("rustup");` -- there is no `rustup` there. It's in `/usr/bin/rustup` -- this may differ probably from distro to distro (not to mention other platforms).

**Comment by pwoolcoc** [comment]
@jmatraszek ah, that makes sense! I was just about to publish a new version with some better error messages, to hopefully narrow this down. I will get to work on a fix for you problem right away, hopefully I'll have something for you in a little while.

**Comment by pwoolcoc** [comment]
@jmatraszek I just published 0.1.1 to crates.io, so you can `cargo install -f cargo-shell` and that should upgrade your shell. Let me know if this doesn't fix the problem, or if there are other problems.

**Comment by jmatraszek** [comment]
Works! Thanks!
